src/Testing/BaseMutatorTestCase.php

```diff
@@ -33,21 +33,19 @@
 
 declare(strict_types=1);
 
-namespace Infection\Tests\Mutator;
+namespace Infection\Testing;
 
+use function array_flip;
```

Comment:
there is more changes that I would do to this class since we make it public (could be done before?). Since I can't comment the non-changed lines I'll add them here:

- inline the type for `#mutator`
- what was the TODO about in the `setUp()`?

Reply:
I don't know, it was created by you.

We shouldn't create TODOs without the ticket anymore in the future